openfoam-interface/internal-interface/wrapper/simphonyInterface.C

```diff
@@ -945,6 +943,9 @@ void foam_modifyNumerics(std::string name, std::string fvSch, std::string fvSol,
         cDIS()
     );
 
+    TimeMod* t = (TimeMod*)(runTimes[name]);
```

Comment:
@nathanfranklin, you are right, but I confused you when I told you about polymorphism. Actually, I'm doing a non-polymorphic inheritance from the base OpenFoam's class "Time". Therefore, and because the only reason to develop the new class TimeMod is to extend the functionality by addition of one function, we can go for static_cast: it's faster and we can treat the memory as if it belonged the type being casted to, because the derived type does not include any additional attribute (new members don't contribute to sizeof()).
Finally, this static_cast is done only when is needed (simphonyInterface.C : 946)